pxemgr/iputil.go

```diff
@@ -16,18 +19,10 @@ func incIP(ip net.IP) net.IP {
 
 // ip less or equal
 func ipLessThanOrEqual(ip net.IP, upperBound net.IP) bool {
-	if ip[3] < upperBound[3] {
-		return true
-	}
-	if ip[2] < upperBound[2] {
-		return true
-	}
-	if ip[1] < upperBound[1] {
-		return true
-	}
-	if ip[0] <= upperBound[0] {
-		return true
-	}
```

Comment:
this actually never worked :p no idea who wrote this code but the test file was working becasue there was only the cases that were suppsoe to be `true` and the function always returned `true` no matter what input